argocd/app-of-apps/Chart.yaml

```diff
@@ -2,3 +2,5 @@
 apiVersion: "v2"
 name: "app-of-apps"
 version: "0"
+description: |
+  This chart and its subcharts install an app, as ArgoCD application.
```

Comment:
I don't think that's precise enough on the role of this app. @mcanevet how would you define it?

Reply:
Well, I guess "App of apps" is a common description for that. It's not a Camptocamp'ism as it's a common pattern described in ArgoCD's documentation.

Reply:
updates following suggestions